keyboards/cannonkeys/satisfaction75/config.h

```diff
@@ -38,6 +38,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define ENCODERS_PAD_A { B9 }
 #define ENCODERS_PAD_B { B8 }
 
+#define ENCODER_DIRECTION_FLIP
```

Comment:
This line may not be needed. 

As noroads pointed out, it depends on the other PR, which flips the direction (so that "clockwise == true" means that YES, that it handles clockwise)

Another option is swapping the pads, as that will change the direction.

Reply:
If I recall correctly the directions are flipped between mac and windows

Reply:
Never mind, volume up is clockwise for both mac and windows